### PR TITLE
Homepage and Styles

### PR DESCRIPTION
**Pull request by raminka13**
This branch was created to develop the homepage (index.html) and set up the repository with tools to help us keep the code clean and efficient.
1. HTML-CSS linters (linters.yml) creation in .github/workflows.
2. Added a .gitignore file to avoid pushing unnecessary files to the repository.
3. Added a more informative Readme file with information about this project.
4. Added our homepage file (index.html) and the Styles file (style.css).
5. Extra commits to clean up code.

### CODE REVIEW

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @ababaug , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [x] Nice work. Please as part of the requirement, kindly add `node_modules` to your .gitignore file
- [x] Also, kindly give a meaningful pull request title and add a pull request description 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,25 @@
-# HelloWorld
\ No newline at end of file
+# Hello World
```
Comment (Ismailco) [inline README.md:1]: - [x] Your pull request should have a summary describing introduced changes in more detail, please include the changes in the PR description

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @raminka13,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (raminka13) [reply README.md:1]: Hi @Ismailco,
The description is the first comment of the PR?. I'll summarize every step in the process.

Comment (Ismailco) [reply README.md:1]: @raminka13, Yes, the first comment of the PR. you can add more information on what you did in this PR

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @raminka13 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.

- [x] please next time try to use Present tense in commit messages like Add instead of Added

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**